docs/versioned_docs/version-zenith/quickstart/822194-daggerize.mdx

```diff
@@ -0,0 +1,246 @@
+
+### Install a dependency
+
+Now that your project is a Dagger Module, you can install other modules into it, as dependencies. A dependency is just a Dagger Module installed into another. As an example, install the `hello` module from earlier:
+
+```shell
+dagger install github.com/shykes/daggerverse/hello
+```
+
+Once the command completes, look at the `dagger.json` file again. You should see that a new dependency has been added, similar to that shown below:
+
+```json
+{
+  "name": "greetings-api",
+  "dependencies": [
+    {
+      "name": "hello",
+      "source": "github.com/shykes/daggerverse/hello@7884cbc1941f85a7c714fc7f1bcc5c148842c21e"
+    }
+  ],
+  "engineVersion": "v0.9.11"
+}
+```
+
+:::note
+The exact Git commit for the module version is recorded in `dagger.json`. Dagger enforces "version pinning", which guarantees that the module version you install is the one you'll be using.
+:::
+
+Go ahead and commit this change as well:
+
+```shell
+git add dagger.json
+git commit -m 'dagger install github.com/shykes/daggerverse/hello'
+```
+
+### Use the installed module
+
+4. Now that your module has its first dependency installed, you can start using it! To do so, simply refer to it by its short name - in this case, `hello`.
+
+List available functions:
+
+```shell
+dagger -m hello functions
+```
+
+You should see a familiar result:
+
+```shell
+Name    Description
+hello   Say hello to the world!
+```
+
+Call a function:
+
+```shell
+dagger -m hello call hello
+```
+
+Again, the output should be familiar:
+
+```shell
+hello, world!
+```
+
+### Daggerize a Go build
+
+The `hello` module is a good example to start with, but it doesn't have any functions to work with a Go project. To Daggerize the Go build for this example project, install the Go builder module from a previous section as another dependency:
+
+```shell
+dagger install github.com/kpenfound/dagger-modules/golang@v0.1.3
+```
+
+Then, call the `Build()` function from the Go builder module to build the project and chain other function calls to export the built binary to the host:
+
+```shell
+dagger -m golang --proj . call \
+  # call the go builder function
+  build --os=$(uname -s) --arch=$(uname -m) --args . \
+  # select the binary file produced by the build
+  file --path=./greetings-api \
+  # export
+  export --path=./api-server
+```
+
+Test the newly-built binary by starting the service and sending it a request:
+
+```shell
+./api-server &
+curl localhost:8080
+```
+
+You should see the following output:
+
+```shell
+got / request from 127.0.0.1:51418
+{"greeting":"Hello Kubecon!"}
+```
+
+At this point, you've partly Daggerized your project, to the extent of building a binary with Dagger. Next, how about containerizing and publishing it?
+
+An easy way to do this is to first add the Wolfi container builder module from earlier as another dependency:
+
+```shell
+dagger install github.com/shykes/daggerverse/wolfi@7884cbc1941f85a7c714fc7f1bcc5c148842c21e
```

Comment:
@shykes I've used the hash here but maybe we could use a version number instead?